frontend/src/main/utils/useBackend.js

```diff
@@ -33,7 +33,7 @@ export function useBackend(queryKey, axiosParameters, initialData) {
             return response.data;
         } catch (e) {
             const errorMessage = `Error communicating with backend via ${axiosParameters.method} on ${axiosParameters.url}`;
-            toast(errorMessage);
+            
```

Comment:
is there a reason why we're removing this line? I don't think we should remove this as it's helpful for figuring out when things go wrong

Reply:
I remove this because I don't want an error message if I did not logged in. And also this error message was not used in our later implementations. And this is the easiest and fastest way to remove the error toast if not logged in. So I directly remove it.

Reply:
Yeah, but we NEED that error message when we ARE logged in.

So this isn't the right way to implement this.